turbopack/crates/turbopack-ecmascript/src/minify.rs

```diff
@@ -43,11 +43,13 @@ pub fn minify(code: Code, source_maps: bool, mangle: Option<MangleType>) -> Resu
     let (src, mut src_map_buf) = {
         let fm = cm.new_source_file(FileName::Anon.into(), source_code);
 
+        let comments = SingleThreadedComments::default();
```

Comment:
```suggestion
        // Collect all comments and pass to the minifier so that `PURE` comments are respected.
        let comments = SingleThreadedComments::default();
```